README.md

```diff
@@ -550,6 +535,7 @@ The fields are as follows:
 * `name`: The name of the operation if exists.
 * `arguments`: An array of [arguments](#arguments) for the operation.
 * `extAttrs`: An array of [extended attributes](#extended-attributes).
+* `parent`: The container of this type.
```

Comment:
Maybe: 

```suggestion
* `parent`: The container of this type as an Object.
```

Or anything else that might aid in the usage. I guess they are of whatever parent type. I guess this comment applies to the rest of the documentation.